### PR TITLE
Noassignees

### PR DESCRIPTION
**Pull request by SimonLab**
## to be merged after #70 
see #71 
add rule when assignee is removed from an issue but the `in-progress` label is still on.

### CODE REVIEW

File: web/controllers/rules/issue/noassignees.ex [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      }
+    else
+      nil
+    end
+  end
+
+  defp error_message(login) do
+    """
+    @#{login} the issue doesn't have anymore assignees but is still `in-progress`.
+    Please assign a user to this issue or remove the `in-progress` label.
```
Comment (SimonLab) [inline web/controllers/rules/issue/noassignees.ex:27]: The message error might not be the best one, any suggestions?

Comment (nelsonic) [reply web/controllers/rules/issue/noassignees.ex:27]: @SimonLab we should put all error messages in a central place so they can easily be edited. (_doesn't have to be done now, but in future..._)

For now we should update it to:
```
"""
:warning: @#{login} issue has **no assignee** but is still `in-progress`. Who is working on it?
 Please assign a user to this issue or remove the `in-progress` label. Thanks! :heart:
"""
```

**Review by nelsonic — APPROVED** [review]
@SimonLab looks good. agree copy should be updated on error/warning message.
otherwise happy to merge. 👍

**Review by samhstn — APPROVED** [review]
I agree that we should have a list of all errors in one place.

Looks good 👍

**Comment by SimonLab** [comment]
At the moment the messages are linked to their rule. Agree that we need to create a config file for the rules. I'd like to allow users to create their own rule for each of their repositories so it will make sense to refactor the error message with this feature

**Pull request merged**